conf/deployment/rosa/ms_environment.yaml

```diff
@@ -0,0 +1,5 @@
+ENV_DATA:
```

Comment:
I recommend to rename this file to `ms_production_environment.yaml` so that if someone sees provided config parameters in jenkins or cli then it is clear that this file sets production env type.

Other than that it looks ok.

Reply:
Renamed File to `ms_production_env.yam` and move this to common config paths `conf/ocs_ci`